src/main/java/no/nav/fo/veilarbvedtaksstotte/utils/OppfolgingUtils.java

```diff
@@ -0,0 +1,22 @@
+package no.nav.fo.veilarbvedtaksstotte.utils;
+
+import no.nav.fo.veilarbvedtaksstotte.domain.OppfolgingPeriodeDTO;
+
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OppfolgingUtils {
+
+    public static boolean erSykmeldtUtenArbeidsgiver(String servicegruppe) {
+        return servicegruppe.equals("VURDU");
+    }
+
+    public static Date getOppfolgingStartDato(List<OppfolgingPeriodeDTO> oppfolgingPerioder) {
+        return oppfolgingPerioder.stream()
+                .filter(oppfolgingPeriode -> oppfolgingPeriode.getSluttDato() != null)
+                .map(OppfolgingPeriodeDTO::getStartDato)
+                .collect(Collectors.toList())
```

Comment:
Hva med `.findFirst().orElse(null)` i stede for `collect`, slik at man ikke risikerer `IndexOutOfBoundsException` på `get(0)`?